src/openrct2/network/network.cpp

```diff
@@ -784,6 +785,9 @@ void Network::LoadGroups()
 		}
 		json_decref(json);
 	}
+
+	// Host group should always contain all permissions.
+	group_list.at(0)->ActionsAllowed.fill(0xFF);
```

Comment:
Hmm, yes, indeed. In that case I'd like to see this value have some name, like `SERVER_GROUP`.